src/mediacatch_s2t/uploader.py

```diff
@@ -31,6 +47,13 @@ def __init__(self, file, api_key, language='da'):
     def _is_file_exist(self):
         return pathlib.Path(self.file).is_file()
 
+    def is_multipart_upload(self) -> bool:
+        if self._is_file_exist():
+            filesize = os.path.getsize(self.file)
+            if filesize > CHUNK_SIZE_MIN:
```

Comment:
Please see: https://docs.aws.amazon.com/en_us/AmazonS3/latest/userguide/qfacts.html

There are requirements for multipart uploads